content/python/concepts/iterators/terms/iter/iter.md

```diff
@@ -60,26 +57,45 @@ The following example applies the optional `sentinel` parameter through a callab
 
 ```py
 class codeNinjaCount:
```

Comment:
The example as it was worked, but I think we'd like to replace it with a completely new example, as codeNinja refers to another site.